src/storage/file_io.c

```diff
@@ -2072,6 +2073,9 @@ fileio_create (THREAD_ENTRY * thread_p, const char *db_full_name_p, const char *
   int o_sync;
 #endif /* WINDOWS */
 
+  // todo: remove me
+  er_print_callstack (ARG_FILE_LINE, "Create file %s \n", db_full_name_p);
```

Comment:
Do you want to keep it?

Reply:
No, I will remove it before committing patch